src/amo/components/App/styles.scss

```diff
@@ -67,6 +67,10 @@ body {
   max-width: $max-content-width;
 }
 
+.App-content-wrapper-homepage {
+  margin: 0 auto;
```

Comment:
Instead of special casing the home page, you could remove this top-margin everywhere and wrap all other pages in a new `<Subpage>` component that re-introduces the margin. Just a thought. 

If you go that direction then you'd want to test *all* pages including some hidden ones like the error pages: http://localhost:3000/en-US/firefox/simulate-client-error/

Reply:
I'm not sure I follow entirely. Would this mean adding `Subpage` to all page-level components, or just adding it into `App`? The former seems like a lot of work just to allow a change for a single page (the home page).

Reply:
I wasn't sure if this was still a suggestion, based on your response at https://github.com/mozilla/addons-frontend/pull/8662#issuecomment-534764251, so I haven't made any changes for this in the upcoming commit.